lib/errata/errata_requests.py

```diff
@@ -189,10 +220,33 @@ def format_advisory_data(advisory_data):
 
     final_response["bugs"] = bug_details
     bug_summary_array = []
+    for key in jira_issue_summary:
+        if total_jira_issues == 0:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": 0
+            })
+        else:
+            bug_summary_array.append({
+                "bug_status": key,
+                "count": jira_issue_summary[key],
+                "percent": round((jira_issue_summary[key] / total_jira_issues) * 100, 2)
```

Comment:
Can I know why we are sending back the percentage? I'm not sure if I see them on the front end, unless I'm missing something? Similarly in Bugzilla as well

Cause we just need the count right?

Reply:
ah this one should not be there, i had some issues reaching the API so I was trying to send all paramaters in documentations for testing, I will remove it.